lib/token-assert.js

```diff
@@ -38,6 +38,21 @@ TokenAssert.prototype.whitespaceBetween = function(options) {
                 column: token.loc.end.column
             });
         }
+    } else if (options.hasOwnProperty('maximumSpaces')) {
```

Comment:
Ah, nvm. Fell into the same trap. That is a different function entirely. I will still rename to `atLeastSpaces` for semi-consistency but whitespace isn't necessarily spaces only.